website/src/components/a11y-filter.js

```diff
@@ -4,7 +4,12 @@ import { jsx, useBrand } from '@westpac/core';
 import Select from 'react-select';
 
 const SVGFilters = () => (
-	<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" version="1.1">
+	<svg
+		aria-hidden="true"
+		style={{ display: 'none' }}
```

Comment:
did you cross browser check this? I think adding `display:none` to svgs breaks safari or firefox?

Reply:
I did not /honesty

Reply:
Dom it pains me to say it but you are correct!

Reply:
Fix on the way